trajopt_ext/osqp_eigen/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>osqp_eigen</name>
+  <version>0.1.0</version>
+  <description>Downloads and builds the OSQPEigen Library</description>
+  <maintainer email="levi.armstrong@swri.org">Levi Armstrong</maintainer>
+  <license>Unknown</license>
```

Comment:
Missing license.

Reply:
I added it as LGPL to be the same as OSQPEigen itself and avoid confusion.